### PR TITLE
Update docs for Liveness and Readiness paths in Boot 2.x+

### LINKED ISSUE #273: Liveness and Readiness probes fail with 404 for Boot 2.0 Apps
As a user, I'm trying to deploy a partitioned stream onto K8S, but the streaming app fails to deploy.

**Observations:**
- The pods continue to crash and restart
- Turns out the k8s-deployer is defaulting to `/info` and `/health` for readiness and liveness probes respectively; however, in Boot 2.0, it is changed to `/actuator/info` and `/actuator/health`.
- Correcting the probe paths similar to the following resolved the problem.

```
stream deploy --name foo --properties "app.fruits1.producer.partitionKeyExpression=payload.id,deployer.appender1.count=3,deployer.websocket.kubernetes.createLoadBalancer=true,deployer.fruits1.kubernetes.readinessProbePath=/actuator/info,deployer.appender1.kubernetes.readinessProbePath=/actuator/info,deployer.fruits1.kubernetes.livenessProbePath=/actuator/health,deployer.appender1.kubernetes.livenessProbePath=/actuator/health"
```

We would have to document it in the reference guide.

### PR DESCRIPTION
**Pull request by chrisjs**
Fixes #273

### CODE REVIEW

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -735,18 +735,31 @@ This overrides the JVM memory setting for the desired <app> (just replace <app>
 
 === Liveness and Readiness Probes
 
-The _liveness_ and _readiness_ probes are using the _paths_ `\health` and `\info` respectively. They use a _delay_ of 10 for both and a _period_ of 60 and 10 respectively. You can chage these defaults when you deploy by using deployer properties.
+The _liveness_ and _readiness_ probes are using the _paths_ `/health` and `/info` respectively. They use a _delay_ of 10 for both and a _period_ of 60 and 10 respectively. You can chage these defaults when you deploy by using deployer properties.
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc]: perhaps: "when you deploy the stream by using ..."

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc [hunk 2]
```diff
@@ -735,18 +735,31 @@ This overrides the JVM memory setting for the desired <app> (just replace <app>
...
+deployer.<app>.kubernetes.livenessProbePath=/health
 deployer.<app>.kubernetes.livenessProbeDelay=120
 deployer.<app>.kubernetes.livenessProbePeriod=20
 ```
 
 Similarly, swap _liveness_ for _readiness_ to override the default readiness settings.
 
+[NOTE]
+====
+If your application is using Spring Boot 2.x+, all endpoints have been moved under `/actuator` by default. Here is an example of configuring Spring Boot 2.x+ _liveness_ and _readiness_ endpoint paths (just replace <app> with the name of your app):
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc]: Here's a suggestion:
> If you intend to use Spring Boot 2.x+, please note that all Actuator endpoints in Spring Boot 2.x have been moved under `/actuator` path by default. You _must_ adjust the liveness and readiness probe paths to the new defaults. Here is an example of configuring Spring Boot 2.x+ _liveness_ and _readiness_ endpoint paths (just replace <app> with the name of your app):

**Review by sabbyanandan — DISMISSED** [review]
Left a few simple change suggestions. Looks good o'wise.

**Comment by chrisjs** [comment]
merged in baab53

**Pull request closed without merging**